status-im/status-react
# [ISSUE #1840] Migrated to FlatList

addresses #1840

### Summary:

Migrated all list components (deprecated in recent RN releases) to new FlatList component.

### Steps to test:

- Sanity check status, especially all `lists`

Test opened chat very throughly, including order of messages, if the new messages are added at the correct place and whether :load-more event is still correctly firing once somebody scrolls at the very top of the message list (test by opening chat with more then 20 messages, scroll to the very top and check if all the messages are loaded and rendered in correct order)

status: ready
